composer.json

```diff
@@ -16,14 +16,15 @@
         "symfony/symfony": "2.8.*",
         "doctrine/orm": "^2.4.8",
         "doctrine/doctrine-bundle": "~1.4",
-        "symfony/swiftmai0ler-bundle": "~2.3,>=2.3.10",
+        "symfony/swiftmailer-bundle": "~2.3,>=2.3.10",
         "symfony/monolog-bundle": "^3.0.2",
         "sensio/distribution-bundle": "~5.0",
         "sensio/framework-extra-bundle": "^3.0.2",
         "incenteev/composer-parameter-handler": "~2.0",
         "ob/highcharts-bundle": "^1.5",
         "knplabs/knp-paginator-bundle": "^2.5",
-        "friendsofsymfony/user-bundle": "~2.0@dev"
+        "friendsofsymfony/user-bundle": "~2.0@dev",
+        "doctrine/doctrine-fixtures-bundle": "~2.3"
```

Comment:
What do you think to put doctrine/doctrine-fixtures-bundle in require-dev rather?